software/mk/Makefile.verilog

```diff
@@ -40,18 +40,27 @@ endif
 #######################################
 # Simulator setup
 
+TOOL ?= VCS
```

Comment:
@drichmond better way to deal with this?

Reply:
My response is what we did in replicant. Use platform.mk as a single source of truth for the simulator. 

The downside is that with different makefiles, cleaning is a bit weird. Sometimes you end up with files floating around from the simulator you're not using.

That or have separate targets for VCS and Verilator. simv and simv.verilator or something. 

Since VCS and Verilator are different enough, there's not a good way to handle the two toolchains like Clang and GCC. You could try though (e.g. generate a .daidir etc for Verilator so that you have the same intermediate outputs)